training/index.rst

```diff
@@ -27,6 +35,7 @@ General Training Material
 * `CUDA training at Oakridge (slides and lecture recording) <https://www.olcf.ornl.gov/cuda-training-series/>`_
 * Sheffield `One <http://gpucomputing.shef.ac.uk/education/sheffield_onedaycuda/>`_ or `Two <http://gpucomputing.shef.ac.uk/education/cuda/>`_ Day Introduction to CUDA Course (slides and labs)
 * `Sheffield Parallel Computing with GPUs taught module (slides, labs, lecture recordings) <https://paulrichmond.shef.ac.uk/teaching/COM4521/>`_ 
+* The `Power AI User Group Slack workspace <https://www.poweraiug.org/join/>`_ is very helpful for all sorts of questions about Power AI.
```

Comment:
Site seems to be down at the moment - presume that's a temporary glitch?

Reply:
Hm, still is down. It would have hoped that it would be back up by today. :-/

Reply:
I'll give it one or two more days, and otherwise remove this section.

Reply:
Could ping Ed Edmondson to ask about it?

Reply:
Good point!

Reply:
OK, I contacted him and he forwarded it, and it seems to be resolved now.